tests/unit_tests/cuda/rk_stepper_cuda.cpp

```diff
@@ -13,7 +13,7 @@
 #include "rk_stepper_cuda_kernel.hpp"
 
 TEST(rk_stepper_cuda, bound_state) {
-
+    /*
```

Comment:
This is not yet tested, I assume?

Reply:
This file should be deleted as I removed the `bound_state` function in `rk_stepper`.